AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/AppLovin/Banners/UI/ALDemoSwiftUIBannerView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ALDemoSwiftUIBannerView.swift
+//  AppLovin MAX Demo App - Swift
+//
+//  Created by Matthew Nguyen on 7/31/23.
+//  Copyright © 2023 AppLovin. All rights reserved.
+//
+
+import SwiftUI
+import Adjust
+import AppLovinSDK
+
+@available(iOS 13.0, *)
+struct ALDemoSwiftUIBannerView: View
+{
+    @ObservedObject private var viewModel = ALDemoAdViewSwiftUIViewModel()
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            callbacksTable
+                .frame(maxHeight: .infinity)
+            
+            ALAdViewSwiftUIWrapper(shouldLoadAd: $viewModel.shouldLoadAd,
+                                   adFormat: .banner,
+                                   didLoad: viewModel.adService(_:didLoad:),
+                                   didFailToLoadAdWithError: viewModel.adService(_:didFailToLoadAdWithError:),
+                                   wasDisplayedIn: viewModel.ad(_:wasDisplayedIn:),
+                                   wasHiddenIn: viewModel.ad(_:wasHiddenIn:),
+                                   wasClickedIn: viewModel.ad(_:wasClickedIn:),
+                                   didPresentfullscreenFor: viewModel.ad(_:didPresentFullscreenFor:),
+                                   willDismissFullscreenFor: viewModel.ad(_:willDismissFullscreenFor:),
+                                   didDismissFullscreenFor: viewModel.ad(_:didDismissFullscreenFor:),
+                                   willLeaveApplicationFor: viewModel.ad(_:willLeaveApplicationFor:),
+                                   didReturnToApplicationFor: viewModel.ad(_:didReturnToApplicationFor:),
+                                   didFailToDisplayIn: viewModel.ad(_:didFailToDisplayIn:withError:))
+                .deviceSpecificFrame()
+            
+            Button {
+                viewModel.shouldLoadAd = true
+            } label: {
+                Text("Load")
```

Comment:
Could we make it so the "Load" button is disabled until the ad display callback is received, similar to the other AppLovin banner examples? Same comment for AppLovin MRECs